openeo/rest/connection.py

```diff
@@ -1361,6 +1368,10 @@ def load_stac(
                 prop: build_child_callback(pred, parent_parameters=["value"]) for prop, pred in properties.items()
             }
         cube = self.datacube_from_process(process_id="load_stac", **arguments)
+        try:
+            cube.metadata = metadata_from_stac(url)
+        except Exception:
+            _log.warning("Python client could not read band metadata from URL.")
```

Comment:
make sure to include info about the thrown exception
```suggestion
            _log.warning(f"Failed to extract cube metadata from STAC URL {url}", exc_info=True)
```